app/frontend/src/components/bcgov/BCGovHeader.vue

```diff
@@ -20,12 +20,10 @@
           width="10rem"
         />
       </a>
-      <h1 data-test="btn-header-title" class="font-weight-bold title">
+      <h1 v-if="!formSubmitMode" data-test="btn-header-title" class="font-weight-bold title">
         {{ appTitle }}
       </h1>
-      <v-chip v-if="!hideBetaChip" class="ma-2" color="green" text-color="white">
-        Beta
-      </v-chip>
+      <v-chip v-if="!formSubmitMode" class="ma-2" color="green" text-color="white">Beta</v-chip>
```

Comment:
Color could be "secondary" to use more bcgov style colors (it should be a golden yellow color)